Src/CSharpier.Playground/Startup.cs

```diff
@@ -45,9 +45,9 @@ ILoggerFactory loggerFactory
                 app.UseExceptionHandler("/Error");
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
+                app.UseHttpsRedirection();
```

Comment:
You probably already know this, but you should really get in the habit of doing this kind of thing in a separate PR :small_airplane:

Reply:
I know, especially now that there are more contributers